src/angular/search/filter-bar.component.html.ts

```diff
@@ -0,0 +1,26 @@
+export const filterBarTemplate = `
+<div class="search-bar-container"  [attr.data-tests-id]="testId" [class.active]="searchQuery && searchQuery.length" >
+    <sdc-input class="search-bar-input" 
+               [label]="label"
+               [placeHolder]="placeholder"
+               [debounceTime]="debounceTime"
+               [(value)]="searchQuery"
+               (valueChange)="searchItem($event)"
+               [testId]="testId + '-input'"></sdc-input>
+    <span class="clear-search-x filter-button" *ngIf="searchQuery" (click)="clearSearchQuery()">
+      <svg-icon 
+        [name]="'close'"
+        [mode]="'secondary'"
+        [size]="'small'">
+      </svg-icon>
+    </span>
+    <span class="magnify-button filter-button" *ngIf="!searchQuery">
```

Comment:
we don't need to use the span and the magnify-button. we can use the svg-icon component with background color and shape